Anchal_Mal_DOM/index1.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+
+<head>
+	<title>Marketplace.1</title>
+	<link rel="stylesheet" type="text/css" href="new1.css">
+</head>
+
+<body>
+	<header>
+		<div class = "parent1">
+		<div class = "parent1a">
+			<ul class = "logo">
```

Comment:
use proper naming convention
Use some nesting like 
primary-nav
   primary-nav--logo
this will help in writing the sass in a better way